great_expectations/expectations/core/expect_column_quantile_values_to_be_between.py

```diff
@@ -606,11 +621,15 @@ def _descriptive_quantile_table_renderer(
                     {
                         "content_block_type": "string_template",
                         "string_template": {
-                            "template": quantile_string if quantile_string else f"{quantile:3.2f}",
+                            "template": (
+                                quantile_string if quantile_string else f"{quantile:3.2f}"
+                            ),
                             "tooltip": {
-                                "content": "expect_column_quantile_values_to_be_between \n expect_column_median_to_be_between"  # noqa: E501
-                                if quantile == 0.50  # noqa: PLR2004
-                                else "expect_column_quantile_values_to_be_between"
+                                "content": (
+                                    "expect_column_quantile_values_to_be_between \n expect_column_median_to_be_between"  # noqa: E501
+                                    if quantile == 0.50  # noqa: PLR2004
+                                    else "expect_column_quantile_values_to_be_between"
+                                )
                             },
```

Comment:
Bad merge?

Reply:
why is this one bad? I just adjusted the string format.

Reply:
I wasn't expecting this part of the code to change, but I guess the formatter only fixes files you are touching.